src/test/java/org/verdictdb/jdbc41/VerdictStatementStreamSqlTest.java

```diff
@@ -0,0 +1,414 @@
+package org.verdictdb.jdbc41;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.verdictdb.commons.DatabaseConnectionHelpers;
+import org.verdictdb.connection.*;
+import org.verdictdb.coordinator.ScramblingCoordinator;
+import org.verdictdb.coordinator.SelectQueryCoordinator;
+import org.verdictdb.core.resulthandler.ExecutionResultReader;
+import org.verdictdb.core.scrambling.ScrambleMeta;
+import org.verdictdb.exception.VerdictDBException;
+import org.verdictdb.jdbc41.VerdictConnection;
+import org.verdictdb.jdbc41.VerdictStatement;
+import org.verdictdb.sqlsyntax.MysqlSyntax;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class VerdictStatementStreamSqlTest {
```

Comment:
Can you add a comment to this function? I want to know what this test case aims to achieve (or compare).